_data/simple-icons.json

```
@@ -5751,6 +5751,11 @@
             "hex": "8D6748",
             "source": "https://mochajs.org/"
         },
+        {
+            "title": "Mock Service Worker",
+            "hex": "FF6A33",
+            "source": "https://github.com/mswjs/msw/blob/master/media/msw-logo.svg"
```

As per the [Contributing Guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines), this link should include the exact commit hash.
```suggestion
            "source": "https://github.com/mswjs/msw/blob/5e69e91f79af181a544dea290a7fa4391d8d2ad4/media/msw-logo.svg"
```